src/node.rs

```diff
@@ -40,45 +55,53 @@ impl Default for Sprawl {
 }
 
 impl Sprawl {
+    /// Creates a new [`Sprawl`]
+    ///
+    /// The default capacity of a [`Sprawl`] is 16 nodes.
     pub fn new() -> Self {
         Default::default()
     }
 
+    /// Creates a new [`Sprawl`] that can store `capacity` nodes before reallocation
     pub fn with_capacity(capacity: usize) -> Self {
         Self {
             id: INSTANCE_ALLOCATOR.allocate(),
-            nodes: Allocator::new(),
+            allocator: Allocator::new(),
             nodes_to_ids: new_map_with_capacity(capacity),
             ids_to_nodes: new_map_with_capacity(capacity),
             forest: Forest::with_capacity(capacity),
         }
     }
 
+    /// Allocates memory for a new node, and returns a matching generated [`Node`]
     fn allocate_node(&mut self) -> Node {
-        let local = self.nodes.allocate();
+        let local = self.allocator.allocate();
         Node { instance: self.id, local }
     }
 
+    /// Stores a new
```

Comment:
```suggestion
    /// Stores a new node in the tree
```

Reply:
Or "Inserts a new node"?

Reply:
"adds a new node in the set"?

No idea which is the right one, but it probably should be a complete sentence :)